src/main.js

```diff
@@ -89,9 +89,15 @@ const customLogins = {};
  */
 async function initialize() {
   process.on('uncaughtException', criticalErrorHandler.processUncaughtExceptionHandler.bind(criticalErrorHandler));
-
   global.willAppQuit = false;
 
+  // prevent using a different working directory, which happens on windows running after installation.
```

Comment:
The `chdir` call should probably come after `initializeArgs`. Command-line `--data-dir` resolves relative paths against the cwd on line 149, and there the cwd should still come from the shell.

Maybe move this into `initializeBeforeAppReady` instead? That should still be early enough to address the original issue.